src/common/command_line.cpp

```diff
@@ -90,6 +95,33 @@ bool is_no(const std::string &str)
 	return false;
 }
 
+#ifdef WIN32
+bool get_windows_args(std::vector<std::string>& args, std::vector<char*>& argptrs)
+{
+	int nArgs = 0;
+	LPWSTR* szArgs = CommandLineToArgvW(GetCommandLineW(), &nArgs);
+
+	if(szArgs == nullptr)
+		return false;
+
+	for(int i=0; i < nArgs; i++)
```

Comment:
because you need to touch this file anyway could you please add spaces around the assign operator `int i = 0; ...`